pkgs/applications/audio/gnome-podcasts/default.nix

```
@@ -32,8 +32,9 @@ stdenv.mkDerivation rec {
     hash = "sha256-LPwCYgAFgUMFQZ0i4ldiuGYGMMWcMqYct3/o7eTIhmU=";
   };
 
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
```

```suggestion
    inherit pname version src;
```

To give the tarball a meaningful name.

---

done